catalyst_voices/uikit_example/lib/examples/voices_snackbar_example.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:catalyst_voices/widgets/snackbar/voices_snackbar.dart';
+import 'package:catalyst_voices/widgets/snackbar/voices_snackbar_type.dart';
+import 'package:flutter/material.dart';
+
+class VoicesSnackbarExample extends StatelessWidget {
+  static const String route = '/snackbar-example';
+
+  const VoicesSnackbarExample({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final screenWidth = MediaQuery.sizeOf(context).width;
+
+    return Scaffold(
+      body: Padding(
+        padding: const EdgeInsets.all(32),
+        child: Wrap(
+          spacing: 16,
+          runSpacing: 16,
+          children: [
+            for (final type in VoicesSnackBarType.values)
+              OutlinedButton(
+                onPressed: () {
+                  VoicesSnackBar(
+                    snackBarType: type,
+                    padding: EdgeInsets.only(
+                      bottom: screenWidth / 2,
+                      left: screenWidth / 3,
+                      right: screenWidth / 3,
+                    ),
+                    onLearnMorePressed: () {},
+                    onRefreshPressed: () {},
+                    onOkPressed: () {},
+                    onClosePressed: () {
+                      ScaffoldMessenger.of(context).hideCurrentSnackBar();
```

Comment:
Would be cool if we had `hideCurrentSnackBar` static method on `VoicesSnackBar` so it we're not mixing those two. Inside of course just call `ScaffoldMessenger`

```dart
VoicesSnackBar.hideCurrentSnackBar(context);
```